### DISCUSSION TITLE
port High Carbon Steel

### DISCUSSION THREAD

**Discussion by scarf005** [cataclysmbn/Cataclysm-BN#2493] [General]
# Summary

> any plans to port graded steel? grades like Low Med High are annoying, but being able to use High Carbon Steel for armor and weapons is nice
> 
> by ssh

had an [AMA][ama] on korean roguelike community, and got a DDA port request. the idea is to not port all the grades, but only the high carbon steel.

## related PRs

- https://github.com/CleverRaven/Cataclysm-DDA/pull/53875
- https://github.com/CleverRaven/Cataclysm-DDA/pull/54606
- https://github.com/CleverRaven/Cataclysm-DDA/pull/53962
- https://github.com/CleverRaven/Cataclysm-DDA/pull/54164 
- https://github.com/CleverRaven/Cataclysm-DDA/pull/54377 
- https://github.com/CleverRaven/Cataclysm-DDA/pull/54535 
- https://github.com/CleverRaven/Cataclysm-DDA/pull/54860
- https://github.com/CleverRaven/Cataclysm-DDA/pull/59254
- https://github.com/CleverRaven/Cataclysm-DDA/pull/56326
- https://github.com/CleverRaven/Cataclysm-DDA/pull/60574

**Comment by chaosvolt** [comment]
Depends on if it really gives us any benefit over just having quality items be made of steel of hardsteel. The latter in particular serves the role of "good strong material" and in particular weighs protection values a bit more towards being better for ballistic armor, at the expense of not being widely used by craftable items.
